packages/salesforcedx-vscode-core/package.json

```diff
@@ -303,7 +303,7 @@
         },
         {
           "command": "sfdx.create.manifest",
-          "when": "sfdx:project_opened"
+          "when": "resource =~ /force-app/ && sfdx:project_opened"
```

Comment:
This piece may break functionality for users who use alternate source root paths, e.g. `src` as is the standard [DX@Scale](https://docs.dxatscale.io)

Reply:
because of this piece we are not able to use the command as we are using multiple app folders - none of which is called force-app

Reply:
Hey y'all, thanks for the feedback here! Can one of you log a bug on this in our issues tab so we can dig in and get a fix out soon? FYI on this one @angela-young https://github.com/forcedotcom/salesforcedx-vscode/issues

Reply:
@randi274 done! Opened issue #4310.